### ISSUE TITLE
Improve plot_rotor (bokeh output)

### ISSUE THREAD

**Issue by GabrielBachmann** [ross-rotordynamics/ross#173]
The Rotor object's plot_rotor() method is not giving a good bokeh output, as we can see in the following picture. 

![image](https://user-images.githubusercontent.com/32821252/59566812-72806c80-903b-11e9-9d16-04a7be39c14d.png)

This is the rotor is from this [example](https://github.com/ross-rotordynamics/ross/blob/master/docs/examples/example_05_09_01.ipynb).

**Comment by rodrigomoliveira1** [comment]
It happens when building rotors with few elements (from 2 to 4 elements in total). Because the graphic representation of disks and bearings are calculated according to the shaft element lenght, I suggest doubling the number of elements in this example. It should look better.

**Comment by raphaeltimbo** [comment]
I don't know if the number of elements should have this influence on the rotor plot aesthetics. Maybe we should evaluate if we could calculate bearings/disks proportions based on other things. I don’t remember how the code for this calculation is implemented, maybe it would be too much effort to change the code for a case that will not be happening with ‘real rotors’. We should start making examples with some different rotors and see if this will still be an issue.

**Comment by rodrigomoliveira1** [comment]
The number of elements have influence since the size of disks' marks are calculated base on the length of shaft elements (1/8 of the element size, to be more exact).
Maybe i can illustrate the problem with some images.

This one uses 5 elements (1 shaft element per section). The circle seens large
![1](https://user-images.githubusercontent.com/45969994/60437550-fc583a00-9be4-11e9-8a3b-4941b4340e08.PNG)

This one uses 10 elements (2 shaft element per section).
![2](https://user-images.githubusercontent.com/45969994/60437712-5bb64a00-9be5-11e9-8b77-73e5a1e94051.PNG)

And this one uses 15 elements (3 shaft element per section).
![3](https://user-images.githubusercontent.com/45969994/60437730-6670df00-9be5-11e9-908f-4e38f3dce5cf.PNG)

As we can see, when more elements were added, the proportion to the image got better.
Notice that neither the rotor length does not change, nor the diameters, nor the zoom (in or out).

We can still use the same rotor models for examples. We only need to add a few more elements to get a better aesthetic.

**Comment by raphaeltimbo** [comment]
I have just plotted the rotor of a compressor (disks not included) and I have some comments:
- Almost every element has a slenderness ratio < 1.6. I think we should separate this slenderness ratio analysis from the rotor_plot. We could create a `.check_slenderness_ratio()` method that will return the colored plot and the hover tool could be used to show the slenderness ratio for each element.
![image](https://user-images.githubusercontent.com/18506378/62149653-9984bc00-b2d2-11e9-86fa-9d0c6e7a621d.png)

- Bearings are too small. As discussed before, I think we should try to make them proportional to the diameter of the element in which the bearing is inserted.
![image](https://user-images.githubusercontent.com/18506378/62149880-1f086c00-b2d3-11e9-99e9-56e5d91313f6.png)

**Comment by rodrigomoliveira1** [comment]
I agree with both your statements. I'll try to fix these issues soon, then.

**Comment by rodrigomoliveira1** [comment]
I did some tests here and came to this model:

![bokeh_plot(1)](https://user-images.githubusercontent.com/45969994/62381993-0ee6cb80-b523-11e9-97ff-e0c2ae360281.png)

The proportions are based on the diamater of the element now.

**Comment by raphaeltimbo** [comment]
Here it looks like it is getting the wrong diameter:
![image](https://user-images.githubusercontent.com/18506378/62430546-52dcfa80-b6f4-11e9-8a02-4acc437e8843.png)

Notice that we have two shaft elements in the same node here, so we should get the maximum outer diameter to position the bearings.

**Comment by rodrigomoliveira1** [comment]
Yes, there is this issue and some other modifications I need to do. But I'll do it in another PR, the changes would conflict with PR #232.

**Comment by rodrigomoliveira1** [comment]
In the PR #232 about the `check_slenderness_ratio` method, I kept the attribute "check_sld" because I needed something to diferentiate the HoverTool selection. One Tool is called when using `plot_rotor` and the other one is called when using `check_slenderness_ratio`. But I did remove the SR attribute from the other method by the way.

I couldn't rewrite the HoverTool by calling the bokeh axis (bk_ax).
